machina/templates/machina/forum_feeds/topics_description.html

```diff
@@ -3,4 +3,8 @@
 
 {{ obj.last_post.content.rendered|safe }}
 <br /><br />
-<i>{% trans "Posted by" %} {{ obj.poster|forum_member_display_name }} - {{ obj.created }}</i>
+{% if obj.poster %}
+	<i>{% trans "Posted by" %} {{ obj.poster|forum_member_display_name }} - {{ obj.created }}</i>
+{% else %}
+	<i>{% trans "Posted by" %} {{ obj.last_post.username }} - {{ obj.created }}</i>
```

Comment:
You should use `obj.first_post.username` since the author of the topic is the author of the first post 😉

Reply:
Oh, you are right! Shame on me.